### PR TITLE
Introduce tests to ensure we define abandonseed values for coin we support

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

Introduce tests to ensure we define abandonseed values for coin we support

necessary to fix ETC swap support

doc: https://ledgerhq.atlassian.net/wiki/spaces/WALLETCO/pages/3870425637/How+to+maintain+abandonseed.ts

### ❓ Context

- **Impacted projects**: LLD/LLM 
- **Linked resource(s)**: LIVE-3963 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts [hunk 1]
```diff
@@ -1,4 +1,11 @@
 import invariant from "invariant";
+
+/**
+ * these are either "dead"/"burn" addresses OR "abandon" seed addresses.
+ * These addresses are PUBLIC addresses
+ * We use them for tests and also for dry-run estimations
+ * DO NOT USE AS RECIPIENT OR SIGN TRANSACTIONS INTO THEM
```
Comment (haammar-ledger) [inline libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts:7]: Maybe in this comment -- and for my own knowledge -- you could document how to get an abandon address.
I'm curious to know how you got them for all the different coins.

**Review by haammar-ledger — APPROVED** [review]
No need for test on LLM?

File: libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts (hunk 1 above)
Comment (gre) [reply libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts:7]: i'll provide an internal doc

**Comment by gre** [comment]
@haammar-ledger there are no set up of Jest on the LLM project, i'm not going to start this in this PR. I assumed here LLD would embrace the same coins as LLD and the test added on LLD is just an extra check on a test that already exists mainly on live-common.

**Pull request merged**